cookbook/core/containerization/spot_instances.py

```diff
@@ -1,8 +1,53 @@
 """
 Using Spot Instances
-----------------------
+--------------------
```

Comment:
@kumare3 this is not complete right?  There's a bunch of things to set up right?  Or am I wrong?  Like there needs to be a Spot ASG?  And the ASG needs to be hooked up to EKS?  Are there any settings in propeller that need to change?

Reply:
@cosmicBboy or @samhita-alla Any volunteers to write this out?

Reply:
Will save for later.